aqua/jumpcut/Portfile

```diff
@@ -26,6 +26,8 @@ distname            ${name}_${version}-src
 extract.suffix      .tgz
 
 checksums           md5     8ea757ff5723c397be557618592e49bc \
```

Comment:
The post indicates you're removing MD5 checksums, but this one seems to have survived.

Reply:
I admit I oversimplified the PR description, but the individual commit messages should be more accurate. The commit message for this change does not mention removal of md5. I deliberately left it in, because it is the only surviving checksum type that I did not change; this demonstrates that the file did not change.

Reply:
Sure, but we have largely deprecated its use, so maybe that's not needed?

Reply:
When you see a commit come through that changes every checksum, you are left to wonder if the author of the commit is just updating checksum types, or whether a [stealth update](https://trac.macports.org/wiki/PortfileRecipes#stealth-updates) is occurring. By leaving one checksum unchanged, the ambiguity is resolved.